docs/assets/js/nhs-frontend-js-check.js

```diff
@@ -0,0 +1 @@
+document.body.className = ((document.body.className) ? document.body.className + " js-enabled" : "js-enabled");
```

Comment:
i don't really understand this - why do we want different styles when javascript is disabled?

Reply:
So the NHS Docs state this class is needed to add JS behavior to certain components. https://service-manual.nhs.uk/design-system/styles/page-template

It also says this script should be in the the body tags. I'll update it

Reply:
It's part of the NHS styling, from memory it affects child NHS components that run essentially the same script and change behaviour based on the availability of JS.  For example a component that has hidden text until you "expand" it will have the content always expanded if JS is disabled.